### PR TITLE
Improve `LibreLinkUp Status` plugin: Display elapsed time since last reading + customizable range + refresh button + error handling + refactor

### PR DESCRIPTION
**Pull request by mauriciabad**
Hi, I made this changes for my own usage on the `LibreLinkUp Status` plugin. It can be useful to more people.

I don-t know how to create the UI to let the users input the constants. I just left them hard-coded on the top of the file, at least users will be able to change the code. Feel free to make a second PR adding the UI.

### CODE REVIEW

**Comment by sprak3000** [comment]
@mauriciabad 

Take a look at the [xbar variables documentation](https://xbarapp.com/docs/2021/03/14/variables-in-xbar.html). You create the variables up top in the metadata comment block (e.g., the `VAR_FIRST_PATIENT` comment you edited). The xbar plugin browser handles displaying the UI for people to populate the variables with their values.

Then, your plugin reads those variables from the environment variables. e.g., `first_patient = os.environ.get("VAR_FIRST_PATIENT")` just above your variables. I'll leave this open for a bit to give you a chance to tinker and update.

**Comment by mauriciabad** [comment]
@sprak3000 Okay, I added the variables, and some more changes I needed. I think it's ready 👍🏽.

Screenshot 2023-11-20 at 5 53 03 PM

**Comment by mauriciabad** [comment]
Well, I'd like one more feature, but I think it's not possible. When I unlock my laptop after sleeping it, the plugin still has the old data, and needs to wait one minute to show the new one. I'd like to refresh the plugin when the laptop wakes up. 

And also, the plugin resets its position in the nav bar, maybe it's because I'm editing its code all the time, I don't know... But it's not very important.

File: Lifestyle/librelinkup.1m.py [hunk 1]
```diff
@@ -3,19 +3,30 @@
...
-# <xbar.author>Florian Schlund</xbar.author>
-# <xbar.author.github>FloSchl8</xbar.author.github>
-# <xbar.desc>Gets your GCM from LibreLinkUp: https://librelinkup.com/</xbar.desc>
-# <xbar.dependencies>python3, python requests</xbar.dependencies>
-# <xbar.image>https://i.imgur.com/XUyOT9o.png</xbar.image>
-# <xbar.var>string(VAR_MAIL=""): Your e-mail.</xbar.var>
-# <xbar.var>string(VAR_PASSWORD=""): Your password.</xbar.var>
+# <xbar.version>v2.0</xbar.version>
+# <xbar.author>Maurici Abad</xbar.author>
+# <xbar.author.github>mauriciabad</xbar.author.github>
```
Comment (sprak3000) [inline Lifestyle/librelinkup.1m.py:8]: We try to keep the original author in the metadata along with any contributors.

```suggestion
# <xbar.version>v2.0</xbar.version>
# <xbar.author>Florian Schlund,Maurici Abad</xbar.author>
# <xbar.author.github>FloSchl8,mauriciabad</xbar.author.github>
```

**Review by sprak3000 — DISMISSED** [review]

File: Lifestyle/librelinkup.1m.py (hunk 1 above)
Comment (mauriciabad) [reply Lifestyle/librelinkup.1m.py:8]: Cool, I wasn't sure how to keep both of us :D

**Pull request merged**